apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -1,27 +1,203 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-
+import {
+  Table,
+  TableHeader,
+  TableRow,
+  TableCell,
+  Text,
+  Button,
+  IdentityBadge,
+  Viewport,
+  theme,
+  IconCheck,
+  IconCross,
+  IconFundraising,
+  IconTime,
+  ContextMenu,
+  ContextMenuItem,
+} from '@aragon/ui'
+import { displayCurrency } from '../../utils/helpers'
+import {
+  AverageRewards,
+  formatAvgAmount,
+  RewardDescription,
+  RewardsTable,
+  NarrowList,
+  NarrowListReward,
+  AmountBadge,
+} from './RewardsTables'
 import { Empty } from '../Card'
+import { provideNetwork } from '../../../../../shared/ui'
+
+const averageRewardsTitles = [ 'My Unclaimed Rewards', 'Year to Date', 'Inception to Date' ]
+// TODO: these need to be actually calculated
+const averageRewardsNumbers = [
+  formatAvgAmount(146, '+$', 'green'),
+  formatAvgAmount(19989.88, '$'),
+  formatAvgAmount(799.87, '$'),
+]
+
+const generateOpenDetails = (reward, openDetails) => () => {
+  console.log('calling openDetails on', reward)
+  openDetails(reward)
+}
+
+const translateToken = (token) => {
+  if (token == 0x0) {
+    return 'ETH'
+  }
+}
+
+const MyRewardsWide = ({ claimed, data, openDetails, network }) => (
+  <Table
+    style={{ width: '100%' }}
+    header={
+      <TableRow>
+        <TableHeader key="1" title="Description" />
+        <TableHeader key="2" title="Transaction" />
+        <TableHeader key="3" title={claimed ? 'Status': 'Transaction Date'} />
+        <TableHeader key="4" title="Amount" />
+      </TableRow>
+    }
+  >
+    {data.map((reward, i) => (
+      <ClickableTableRow key={i} onClick={generateOpenDetails(reward, openDetails)}>
+        <TableCell>
+          <RewardDescription>
+            {reward.description}
+          </RewardDescription>
+        </TableCell>
+        <TableCell>
+          <IdentityBadge
+            networkType={network.type}
+            entity={reward.creator}
```

Comment:
There's no creator shown in the spec or the invision wireframes: https://projects.invisionapp.com/d/main#/console/14716432/332905229/preview

I think this column should be removed

Reply:
That's 'transaction' column. Here it shows an eth address, which happened to be creator's address, but it should be changed in integration phase.

Reply:
A tx hash is 32 bytes and is essentially the id for a given transaction. An address is 20 bytes, so I'm pretty sure the Identity badge isn't applicable here.

Reply:
Aw. Of course. TX ID, not address. Fixed!